### PR TITLE
I made some new data

### PR DESCRIPTION
**Pull request by ErwinSputra**
Merge this please ✌️

### CODE REVIEW

**Comment by ErwinSputra** [comment]
Can you guys don't approve this, but request change. I wanna know what gonna happen next

File: logbook.json [hunk 1]
```diff
@@ -27,5 +27,9 @@
   {
     "log": "What do you buyin",
     "by": "arghanovanr"
+  },
+  {
+    "log": "Yesterday, my pull request can't be merge cause conflict file it think. So, i made new one.",
```
Comment (ednagealmira) [inline logbook.json]: i think this part might need some changes.

**Review by ednagealmira — CHANGES_REQUESTED** [review]

File: logbook.json (hunk 1 above)
Comment (ErwinSputra) [reply logbook.json]: Okay, i'll change it right away

**Comment by ErwinSputra** [comment]
I already change some data, approve this please :v

**Review by ErwinSputra — COMMENTED** [review]
I already change some data in this part. Approve this please :v

**Review by ErwinSputra — COMMENTED** [review]
This part exactly

**Review by LordAldi — APPROVED** [review]
mou fainetai oraio

**Review by ednagealmira — APPROVED** [review]
lgtm

**Pull request merged**